CHANGELOG_AUTO.md

```diff
@@ -0,0 +1,657 @@
+# Change Log
+
+## [Unreleased](https://github.com/brigade/overcommit/tree/HEAD)
+
+[Full Changelog](https://github.com/brigade/overcommit/compare/v0.25.0...HEAD)
+
+**Implemented enhancements:**
+
+- Link to hook homepages where available [\#210](https://github.com/brigade/overcommit/issues/210)
+
+**Merged pull requests:**
+
+- Add demo GIF to README.md [\#208](https://github.com/brigade/overcommit/pull/208) ([lencioni](https://github.com/lencioni))
+
+- Add EmptyMessage commit-msg hook [\#207](https://github.com/brigade/overcommit/pull/207) ([jawshooah](https://github.com/jawshooah))
+
+## [v0.25.0](https://github.com/brigade/overcommit/tree/v0.25.0) (2015-05-02)
+
+[Full Changelog](https://github.com/brigade/overcommit/compare/v0.24.0...v0.25.0)
```

Comment:
In my case I'm specifically referring to bullet points with blank lines between them. Consider the following two lists:

### List without blank lines
- One
- Two
- Three

### List with blank lines
- One

- Two

- Three

I prefer the former to the latter as it takes up less vertical space, but the tool outputs the latter.

I would say that supporting arbitrary templates would solve this problem (but it is a pretty minor issue).